src/Microsoft.Health.Dicom.Client/DicomWebClient.cs

```diff
@@ -396,6 +424,9 @@ private static MediaTypeWithQualityHeaderValue CreateMultipartMediaTypeHeader(st
         private static Uri GenerateStoreRequestUri(string studyInstanceUid)
             => new Uri(string.Format(DicomWebConstants.BaseStudyUriFormat, studyInstanceUid), UriKind.Relative);
 
+        private static Uri GenerateCustomTagUri(string tagPath)
+            => new Uri(string.Format(DicomWebConstants.BaseCustomTagUriFormat, tagPath), UriKind.Relative);
```

Comment:
If you save the base `Uri` within the class, then you can use the ctor [`Uri(Uri, string)`](https://docs.microsoft.com/en-us/dotnet/api/system.uri.-ctor?view=net-5.0#System_Uri__ctor_System_Uri_System_String_) for combining